pkg/tcpip/stack/ndp.go

```diff
@@ -251,6 +255,18 @@ type NDPDispatcher interface {
 	OnDHCPv6Configuration(tcpip.NICID, DHCPv6ConfigurationFromNDPRA)
 }
 
+// NDPRouterConfiguration is the NDP configuration for Router Advertisements
+type NDPRouterConfiguration struct {
+	RouterLifetime                     time.Duration
+	CurrHopLimit                       uint8
+	Prefix                             tcpip.Address
```

Comment:
An RA can hold 0, 1 or more prefix information options with their own flags and lifetimes.

Lets support that use case :)